### PR TITLE
Address a couple lingering issues higlighted by Lim

### PR DESCRIPTION
**Pull request by davidlmobley**
This:
- Adds Lim acknowledgment
- Clarifies what conditional convergence means
- Explain why 1-4 interactions typically reduced

That finalizes the issues Victoria Lim raised.

### CODE REVIEW

**Comment by davidlmobley** [comment]
@hmayes or @JIMonroe can you review?

(And feel free to merge if you like, though at the moment I can't get the PDF to compile for me for some reason. Doesn't seem to have anything to do with these changes, I'm getting an obscure error about `./basic_training.tex:727: pdfTeX error (ext4): \pdfendlink ended up in different nesting level than \pdfstartlink.` This happens every time I compile with references so it must have to do with bibliography somehow.`

**Comment by davidlmobley** [comment]
Apparently the compilation issue was caused by a URL running into a page break so I fixed it by adjusting the wording slightly to push the URL away from the page break.

**Comment by davidlmobley** [comment]
OK, I just need someone to do a super quick check of this so I can merge. I have some more changes to make (from the peer reviews) and ideally I'd bring them in separately.

I'll start a new branch in the meantime.

File: paper/basic_training.tex [hunk 1]
```diff
@@ -1030,15 +1031,15 @@ \subsection{Long range electrostatics}
...
-While the number of type of bonded interactions remain unchanged during an MD simulation, the strength and importance of non-bonded interactions varies substantially as a simulation proceeds. 
+While the number of type of bonded interactions remain unchanged during an MD simulation, the strength and importance of non-bonded interactions varies substantially as a simulation proceeds.
 
 Additionally, Coulombic interactions fall off only very slowly with distance, as $r^{-1} $, further complicating handling of non-bonded interactions in two different ways.
-First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order of operations~\cite{LeachBook}, meaning we must exercise extreme care.
+First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order in which it is evaluated~\cite{LeachBook}, meaning we must exercise extreme care or the result will be ambiguous.
```
Comment (justinGilmer) [inline paper/basic_training.tex]:
```suggestion
First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order \emph{in which it is evaluated}~\cite{LeachBook}, meaning we must exercise extreme care or the result will be ambiguous.
```

Seems like a pretty important point to make. Might be worthwhile to italicize it.

File: paper/basic_training.tex [hunk 2]
```diff
@@ -432,11 +432,12 @@ \subsubsection{Key concepts}
...
+However, many all-atom energy functions commonly used in biomolecular simulations retain only \emph{partial} nonbonded interactions between terminal atoms involved in a torsion.
+The atoms involved in a torsion, if numbered beginning with 1, would be 1, 2, 3, and 4, so the terminal atoms could be called atoms 1 and 4, and nonbonded interactions between such atoms are called ``1-4 interactions''.
 These interactions are often present but reduced, though the exact amount of reduction differs by the energy function or force field family.
 For example, the AMBER family force fields usually reduce 1-4 electrostatics to $\frac{1}{1.2}$ of their original value, and 1-4 Lennard-Jones interactions to $\frac{1}{2}$ of their original value.
 1-4 interactions are essentially considered the borderline between the bonded and non-bonded regions.
+Because these are short-range interactions, they are potentially quite strong and there is potentially a risk of them overwhelming longer-range interactions, hence their typical reduction.
```
Comment (justinGilmer) [inline paper/basic_training.tex]: The use of `potentially` twice so close together sounds a bit awkward.
```suggestion
These short-range interactions can be quite strong and there is potentially a risk of them overwhelming longer-range interactions, hence their typical reduction.
```

File: paper/basic_training.tex (hunk 1 above)
Comment (davidlmobley) [reply paper/basic_training.tex]: OK, I'll fix in my other PR (Heather merged this one).

File: paper/basic_training.tex (hunk 2 above)
Comment (davidlmobley) [reply paper/basic_training.tex]: Good idea, I'll get that.

**Pull request merged**